test/unit/mdc-list/foundation.test.js

```diff
@@ -843,13 +852,41 @@ test('#setSelectedIndex removes selected/activated class name and sets aria-sele
   const {foundation, mockAdapter} = setupTest();
 
   td.when(mockAdapter.getListItemCount()).thenReturn(4);
+  td.when(mockAdapter.getAttributeForElementIndex(2, strings.ARIA_CURRENT)).thenReturn(null);
+  td.when(mockAdapter.getAttributeForElementIndex(3, strings.ARIA_CURRENT)).thenReturn(null);
   foundation.setSelectedIndex(2);
 
   foundation.setSelectedIndex(3);
   td.verify(mockAdapter.removeClassForElementIndex(2, cssClasses.LIST_ITEM_SELECTED_CLASS), {times: 1});
   td.verify(mockAdapter.setAttributeForElementIndex(2, strings.ARIA_SELECTED, 'false'), {times: 1});
 });
 
+test('#setSelectedIndex sets aria-current="false" to previously selected index and sets aria-current without any token'
```

Comment:
Do we really need both of these tests? Should just the `'page'` test suffice?

Also, should we have a test somewhere for verifying that aria-current vs. aria-selected is properly detected on initialization?